app/src/main/java/com/example/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -19,9 +21,14 @@ public ParkingView(Activity activity, ActivityMainBinding binding) {
 
     @Override
     public void showParkingsAvailable(int parkingsAvailable) {
-        Context context = getContext();
-        if (context != null) {
-            Toast.makeText(context, context.getString(R.string.toast_main_parking_size, parkingsAvailable), Toast.LENGTH_LONG).show();
+        FragmentManager fragmentManager = getFragmentManager();
+        DialogFragmentParkingLots editNameDialogFragment = new DialogFragmentParkingLots();
+        editNameDialogFragment.show(fragmentManager, "DIALOG_FRAGMENT_TAG");
```

Comment:
A little error, you need to use the constant DIALOG_FRAGMENT_TAG without "" , because you continue used a hardcode. Put DIALOG_FRAGMENT_TAG instead of "DIALOG_FRAGMENT_TAG"

Reply:
done